serverless/jobs/how-to/create-job-from-external-registry.mdx

```diff
@@ -0,0 +1,50 @@
+---
+meta:
+  title: How to create a job from an external container registry
+  description: This page explains how to deploy a serverless job from an external registry
+content:
+  h1: How to create a job from an external container registry
+  paragraph: This page explains how to deploy a serverless job from an external registry
+tags: deploy create job external registry docker image
+dates:
+  validation: 2023-11-09
+  posted: 2023-11-09
+categories:
+  - serverless
+  - jobs
+---
+
+Scaleway allows you to create jobs from several container [registries](/containers/container-registry/concepts/#registry), including external public registries, such as Docker Hub, AWS Container registries, GitLab container registry, etc.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - Your container image is available without authentication
+</Message>
+
+1. Click **Jobs** in the **Serverless** section of the side menu. The jobs page displays.
+
+2. Click **Create job**. The job creation wizard displays.
+
+3. Complete the following steps in the wizard:
+    - Select the **External** Container Registry.
```

Comment:
```suggestion
    - Select the **External** container registry.
```

not sure about this one, but I guess it should be lowercase as it refers to an external registry